CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Changelog
 
-## [cryptoki-sys-0.1.4](https://github.com/parallaxsecond/rust-cryptoki/tree/cryptoki-sys-0.1.4) (2022-08-03)
+## [cryptoki-0.4.0](https://github.com/parallaxsecond/rust-cryptoki/tree/cryptoki-0.4.0) (2022-09-07)
```

Comment:
I can't easily tell if the contents of this CHANGELOG for 0.4.0 are correct, it's a bit difficult to see what changed via the GitHub web interface since the 0.3.0 release, especially as it concerns only a subtree in the repo).

Reply:
Yeah, it's not exactly easy to follow what changes/PRs were for `cryptoki`, and which were for `cryptoki-sys`. I'm thinking of adding a label for issues and PRs to at least make it easier to comb through past stuff. Already made a `breaking-change` one, will retroactively apply it.